e2e/cp_test.go

```diff
@@ -3346,3 +3347,73 @@ func TestCopyRawModeAllowDestinationWithoutPrefix(t *testing.T) {
 		t.Errorf("testfile*.txt not exist in S3 bucket %v\n", dst)
 	}
 }
+
+func TestCopyFalseCredentialsExitCode1(t *testing.T) {
+	t.Parallel()
+
+	const bucket = "bucket"
+
+	s3client, s5cmd, cleanup := setup(t)
+	defer cleanup()
+
+	createBucket(t, s3client, bucket)
+
+	folderLayout := []fs.PathOp{
```

Comment:
Please use a simpler test case. This might be an artifact of C/P but it is confusing to see nested files while we're only testing empty credentials scenario.

Reply:
deleted the complex file structures.